xmtp_mls/src/groups/group_mutable_metadata.rs

```diff
@@ -45,7 +45,7 @@ pub enum MetadataField {
 }
 
 impl MetadataField {
-    fn as_str(self) -> &'static str {
+    pub fn as_str(&self) -> &'static str {
```

Comment:
```suggestion
    pub const fn as_str(&self) -> &'static str {
```

might as well make it `const` as well

Reply:
updated 519e0c30e0a7a4df1fdeea91c04dcba0422e3032